code/Model/Indexer/Algolia.php

```diff
@@ -171,14 +175,6 @@ protected function _registerCatalogProductEvent(Mage_Index_Model_Event $event)
         return $this;
     }
 
-    protected function _isProductComposite($productId)
```

Comment:
I'm not sure we want to remove the method completely. It definitelly makes sense to move the method to one class but I'm worried about such BC break. I am thinking about keeping the method and just call `$this->index->isProductComposite($productId);` from it and trigger `Deprecated` warning? WDYT?